lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -187,6 +176,58 @@ def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: Secur
 
         reuse_identifier
       end
+
+      # Update a milestone for a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The number of the milestone we want to fetch
+      # @param options [Hash] A customizable set of options.
+      # @option options [String] :title A unique title.
+      # @option options [String] :state
+      # @option options [String] :description A meaningful description
+      # @option options [Time] :due_on Set if the milestone has a due date
+      # @return [Milestone] A single milestone object
+      # @see http://developer.github.com/v3/issues/milestones/#update-a-milestone
+      #
+      def update_milestone(repository:, number:, options:)
```

Comment:
Nitpick (that I should have mentioned earlier, sorry): while I think having named parameters is way nicer for most cases, the case of `options:` — especially it being a `Hash` and the last parameter of the Ruby function that could gather additional arbitrary keywords/options — might be better served using the little-known but magic [Hash-splat operator trick that Ruby supports](https://ruby-doc.org/core-2.2.2/doc/syntax/calling_methods_rdoc.html#label-Hash+to+Keyword+Arguments+Conversion):

```suggestion
      def update_milestone(repository:, number:, **options)
```

This will allow the call site to then just provide the options directly as if they were keywords (kinda "flattening the Hash into keyword parameters") without having to wrap those options into `{ … }` curly braces to make them a Hash. So the call site would look like this:

```ruby
# Instead of:
#   github_helper.update_milestone(repository: repository, number: milestone[:number], options: { title: mile_title })
# We'd have:
github_helper.update_milestone(repository: repository, number: milestone[:number], title: mile_title)
# Inside `update_milestone`'s implementation, the variable/parameter `options` would be a Hash equal to `{ title: mile_title }`
```
```ruby
# Instead of:
#   github_helper.update_milestone(repository: repository, number: milestone[:number], options: { state: 'closed' })
# We'd have:
github_helper.update_milestone(repository: repository, number: milestone[:number], state: 'closed')
# Inside `update_milestone`'s implementation, the variable/parameter `options` would be a Hash equal to `{ state: 'closed' }`
```

And thanks to the `**` operator, the `options` variable inside the `update_milestone`'s body would still be a `Hash` like before. But at least that would make the call sites nicer, avoiding them to have an additional level to wrap the options into a Hash explicitly at call site 🙃 

<details><summary>Demo (in the <a href='https://github.com/pry/pry'>pry</a> REPL)</summary>

```ruby
[2] pry(main)> def foo(a:, b:, **options)
[2] pry(main)*   puts "a: #{a.inspect}"
[2] pry(main)*   puts "b: #{b.inspect}"
[2] pry(main)*   puts "options: #{options.inspect}"
[2] pry(main)* end  
=> :foo
[3] pry(main)> foo()
ArgumentError: missing keywords: :a, :b
from (pry):3:in `foo'
[4] pry(main)> foo(a: 1, b: 'hello')
a: 1
b: "hello"
options: {}
=> nil
[5] pry(main)> foo(a: 1, b: 'hello', size: 3, skip: false)
a: 1
b: "hello"
options: {:size=>3, :skip=>false}
```
</details>

This is really a nitpick and not a blocker per se, but I figured that since it's quite a little-known / advanced feature of Ruby, it would still be worth mentioning in case you wanted to adopt it 🙂 

(Same for all the other actions similarly having an `options:` keyword, of course)

Reply:
Perfect! I really like that suggestion, it will make the method calls even cleaner. 😄 

And, as I said on Slack, I believe that will be better to address this on other PR as It is a small change, will be easier to review and track any other improvements like this.